tests/unittests/tests-cbor/tests-cbor.c

```diff
@@ -272,6 +272,16 @@ static void test_byte_string(void)
         TEST_ASSERT(cbor_deserialize_byte_string(&stream, 0, buffer, sizeof(buffer)));
         CBOR_CHECK_DESERIALIZED(input, buffer, EQUAL_STRING);
     }
+
+    cbor_clear(&stream);
+
+    {
+        /* check out buffer too small */
+        const char *input = "a";
+        TEST_ASSERT(cbor_serialize_byte_string(&stream, input));
```

Comment:
missing `unsigned char data[] = {0x41, 0x61};` above and below `CBOR_CHECK_SERIALIZED(stream, data, sizeof(data));`, as test cases before

Reply:
added